notes.md

```diff
@@ -265,6 +265,36 @@ was called Timecode, the `TimestampScale Element` was called TimecodeScale, the
 `TrackTimestampScale Element` was called TrackTimecodeScale and the
 `ReferenceTimestamp Element` was called ReferenceTimeCode.
 
+## Raw And Scaled Timestamps
+
+There are 2 types of Timestamps in Matroska, the raw ones stored in nanoseconds and the scaled timestamps.
+
+The elements with raw timetamps in nanoseconds are:
+
+* `Cluster\Timestamp`; defined in (#timestamp-element)
```

Comment:
This is not in nanoseconds at all. The same goes for CueTime below (and probably also for CueRefTime).

Reply:
You're right. The XML text is correct but this is not.

Nice catch !

Reply:
This was already fixed by a95611e50cd5a847d3faca393f437319a9314139 "merge Raw And Scaled Timestamps and Timestamp Ticks sections".